ts/Core/Chart/ChartOptions.d.ts

```diff
@@ -0,0 +1,165 @@
+/* *
+ *
+ *  (c) 2010-2021 Torstein Honsi
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type { AlignObject } from '../Renderer/AlignObject';
+import type { ButtonRelativeToValue } from '../../Maps/MapNavigationOptions';
+import type AnimationOptions from '../../Core/Animation/AnimationOptions';
+import type Axis from '../Axis/Axis';
+import type Chart from './Chart';
+import type ColorType from '../../Core/Color/ColorType';
+import type CSSObject from '../Renderer/CSSObject';
+import type { HTMLDOMElement } from '../Renderer/DOMElementType';
+import type O from '../Options';
+import type { SeriesTypeOptions } from '../Series/SeriesType';
+import type ShadowOptionsObject from '../Renderer/ShadowOptionsObject';
+import type SVGAttributes from '../Renderer/SVG/SVGAttributes';
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+declare module './Chart/ChartLike'{
+    interface ChartLike {
+        marginRight: ChartOptions['marginRight'];
+        polar: ChartOptions['polar'];
+    }
+}
+
+export interface ChartOptions {
```

Comment:
I would have expect something like `declare module '../OptionsLike' {` for `chart: ChartOptions`.